app/assets/javascripts/analytics/static-tracker.js

```diff
@@ -81,6 +82,33 @@
 
   };
 
+  StaticTracker.prototype.callOnNextPage = function(method, params) {
+    params = params || [];
+
+    if (!$.isArray(params)) {
+      params = [params];
+    }
+
+    if (GOVUK.cookie && typeof this[method] === "function") {
+      params.unshift(method);
+      GOVUK.cookie('analytics_next_page_call', params.join('|'));
+    }
+  };
+
+  StaticTracker.prototype.callMethodRequestedByPreviousPage = function() {
+    if (GOVUK.cookie && GOVUK.cookie('analytics_next_page_call') !== null) {
+      var params = GOVUK.cookie('analytics_next_page_call').split('|'),
```

Comment:
This means that none of the parameters passed to the method can contain a '|' character, which isn't obvious and seems likely to trip us up in future.  Parameters will also be silently converted to strings, which might trip people up (though, this being javascript, often won't be noticed).

Might it be better to limit the split to split only once, and pass the full remaining cookie data to the provided method, so the method can parse it as it wants. Particularly since our only current use just wants a single string parameter.